code/_globalvars/misc.dm

```diff
@@ -11,6 +11,9 @@ GLOBAL_DATUM_INIT(data_core, /datum/datacore, new)
 GLOBAL_VAR_INIT(CELLRATE, 0.002)  // conversion ratio between a watt-tick and kilojoule
 GLOBAL_VAR_INIT(CHARGELEVEL, 0.001) // Cap for how fast cells charge, as a percentage-per-tick (.001 means cellcharge is capped to 1% per second)
 
+GLOBAL_VAR_INIT(total_merits_exchanged, 0)
+GLOBAL_VAR_INIT(hydrogen_stored, 0) // can be -/+, + meaning surplus
```

Comment:
This seems weird as a global var.  I know we currently have 1 outpost but this blocks out something like a refueling ruin or something similar

Reply:
Should I tie this to the outpost then? Might need help figuring that out.

Reply:
It seems like to me it should be attached to the computer.